Oceans global (#139)
subnero1/subnero1.github.io: _posts/2020-10-12-Global-Oceans-2020.md

---
layout: post
title: Subnero at Global Oceans 2020
banner : images/banner-pulse-standard.jpg
date:  2020-10-12
categories: wnc, news
thumbnail: images/pulse-thumbnail-auv2020.jpg
---

Due to the pandemic, the physical Oceans 2020 Singapore conference and exhibition have been postponed to 2024.

> To make sure that we don't miss the exciting new papers and stay up to date with the latest development, the IEEE OES decided to combined Oceans Singapore and US Gulf coast into a single virtual event: [Global Oceans 2020](https://global20.oceansconference.org/)

Reviewer: ```suggestion
To make sure that we don't miss the exciting new papers and stay up to date with the latest development, the IEEE OES decided to combine Oceans Singapore and US Gulf coast into a single virtual event: [Global Oceans 2020](https://global20.oceansconference.org/){:target="_blank"}.
```

> The Subnero team was very pleased to deliver a tutorial about practical underwater software-defined network. The team talked about the fundamentals of underwater communications and networking, and many practical tools and techniques to solve common challenges. In the hands-on sessions, participants were able to try out the demos themselves using the freely available UnetStack networking stack and simulator.

Reviewer: ```suggestion
The Subnero team was very pleased to deliver a tutorial about practical underwater software-defined network. The team talked about the fundamentals of underwater communications and networking, and many practical tools and techniques to solve common challenges. In the hands-on sessions, participants were able to try out the demos themselves using the freely available [UnetStack](https://unetstack.net/){:target="_blank"} networking stack and [simulator](https://unetstack.net/handbook/unet-handbook_part_vi_simulating_underwater_networks.html){:target="_blank"}.
```

> The session took place virtually with a serie of 6 videos:

Reviewer: ```suggestion
The session took place virtually with a series of 6 videos:
```
